editor/editor_file_system.cpp

```diff
@@ -1186,7 +1176,7 @@ void EditorFileSystem::_process_file_system(const ScannedDirectory *p_scan_dir,
 					scan_actions.push_back(ia);
 				}
 
-				if (fc->type.is_empty()) {
+				if (!fc->type.is_empty()) {
```

Comment:
Is this really correct?
In your previous iteration you did `fc->type == StringName()`, now the effect is opposite.